nginx_proxy/README.md

```diff
@@ -0,0 +1,113 @@
+# Hass.io Core Add-on: NGINX Home Assistant SSL proxy
```

Comment:
This should be a separate PR?

Reply:
Yes, sorry. I meant to submit as separate PR but still getting familiar with Github. I thought I reverted this change. Anyways, a separate PR should have been submitted anyways and this change can be canceled.